### PR TITLE
Upstream a bunch of additions and changes to unistd

### PR DESCRIPTION
**Pull request by timwoj**
The [Zeek](https://github.com/zeek/zeek) project recently had a port to Windows contributed by a team at Microsoft. Included in the port was a number of changes to libunistd to add needed methods and fix a few bugs. This is the first in a set of PRs that I'm putting together to upstream as many of these changes back to the main repo as possible. This is the easy set, where it's just additional files and such.

I have other changes that will take a bit of massaging to get ready to PR back to upstream. Those will come later.

### CODE REVIEW

File: unistd/fts.c [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+// Provided by brimdata on 2022/10/19
+// Copyright (c) 2019 Robin.Rowe@CinePaint.org
+// License open source MIT
+
+#include "fts.h"
+#include <stddef.h>
+
+FTS *fts_open(char * const *path_argv, int options, int (*compar)(const FTSENT **, const FTSENT **))
+	{
+	return NULL;
```
Comment (timwoj) [inline unistd/fts.c]: Should these three methods use the `STUB_*` macros?

File: unistd/fts.c [hunk 2]
```diff
@@ -0,0 +1,22 @@
+// fts.c
+// Provided by brimdata on 2022/10/19
+// Copyright (c) 2019 Robin.Rowe@CinePaint.org
```
Comment (timwoj) [inline unistd/fts.c]: This and a few other files had this copyright header already in place when we got them from Microsoft, but they weren't already in the repo. We're fine with assigning the copyright to you here, but I just want to bring attention to it.

**Review by robinrowe — CHANGES_REQUESTED** [review]
Email me at robin.rowe@heroicrobots.com.

Thank you for updates.  I've added a Contributors section to the project README to address some issues, a few minor style things I'd like resolved. Please take a look and update patches.

Robin

File: unistd/fts.c (hunk 2 above)
Comment (robinrowe) [reply unistd/fts.c]: You can't assign copyright if you're not the owner.

Comment (timwoj) [reply unistd/fts.c]: I got ahold of Noah Treuhaft at brimdata, who was the original source of the files. I've fixed all of the copyright attributions for those files to match the requested format.

**Comment by timwoj** [comment]
Alright I think I've caught up with everything you added to the contributor notes. Let me know if there's anything else, or if I should squash everything back into the original commit set again.

**Comment by timwoj** [comment]
Great, thanks. I'll get started on the next set 😃

**Pull request merged**